<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>createPassw</class>
 <widget class="QDialog" name="createPassw">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>533</width>
    <height>220</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>533</width>
    <height>200</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>13</x>
     <y>19</y>
     <width>511</width>
     <height>131</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="labelResource">
      <property name="text">
       <string>Resource*:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="labelDescription">
      <property name="text">
       <string>Description:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="lineResource"/>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineDescription"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="labelPassw1">
      <property name="text">
       <string>Password*:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="linePassw1"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="labelPassw2">
      <property name="text">
       <string>Password once again*:</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="linePassw2"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>511</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="buttonOk">
      <property name="text">
       <string>OK</string>
      </property>
      <property name="icon">
       <iconset resource="../../resource.qrc">
        <normaloff>:/img/add.png</normaloff>:/img/add.png</iconset>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="buttonCancel">
      <property name="text">
       <string>Cancel</string>
      </property>
      <property name="icon">
       <iconset resource="../../resource.qrc">
        <normaloff>:/img/delete.png</normaloff>:/img/delete.png</iconset>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../../resource.qrc"/>
 </resources>
 <connections/>
</ui>
